code/game/gamemodes/events/apc_damage.dm

```diff
@@ -44,3 +44,6 @@
 /proc/is_valid_apc(var/obj/machinery/power/apc/apc)
 	var/area/A = get_area(apc)
 	return !(A && (A.flags & AREA_FLAG_CRITICAL)) && !apc.emagged && isOnShipLevel(apc)
+
```

Comment:
More blank lines. These aren't needed are they?

Reply:
is there a quick and easy way to remove them?

Reply:
Undoing what you did in the file to create them? I guess it doesn't hurt anything it just creates clutter in the diff.